### DISCUSSION TITLE
Risk in schools

### DISCUSSION THREAD

**Discussion by megll** [microCOVID/microCOVID#1044] [General]
Thanks so much for making the Risk Tracker - it’s a great idea. I’m getting nervous about my kid going back to school. I put different school-like conditions into the Calculator (I used "going to work" as the type of scenario) and every time, it rates the risk as “dangerously high” — even with him being vaccinated, wearing a KN95 mask, and the other students wearing masks as well. For younger kids who can’t get the vaccine, in a room full of others too young to get the vaccine, the risk would be even higher. Is there any reason to think the Calculator is overestimating the risk in school conditions ? Otherwise, it seems very risky to be fully opening the schools now while faced with the Delta variant, even with universal masking. Wondering if others have thoughts about this.

**Comment by blanchardjeremy** [comment]
Hi @megll !

You're not alone. This is the top question we've been receiving lately. 

You can see all the related issues that are open here: https://github.com/microCOVID/microCOVID/labels/children%2Fschools

We're in the middle of doing some more in-depth research. Until then, anything we could say would be pretty speculative.
